options/locale/locale_en-US.ini

```diff
@@ -923,12 +923,16 @@ repo_and_org_access = Repository and Organization Access
 permissions_public_only = Public only
 permissions_access_all = All (public, private, and limited)
 permission_not_set = Not set
+permission_not_set_helper = No public access to this unit
```

Comment:
It's not true. If a repo is "public", then "not set" also means "public"

That's why  here I intentionally used "not set" but not "none" or "no access". Not set means there is no extra public access rules for this repo, it follows its own visibility and member permissions.